portals/developer-portal/src/defaultContent/styles/main.css

```
@@ -1,4 +1,6 @@
 @import url("https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100..900;1,100..900&family=Quicksand:wght@300..700&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500;600;700&display=swap");
```

Stylelint (17.13.0)

[error] Expected "url("https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500;600;700&display=swap")" to be ""https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500;600;700&display=swap"" (import-notation)

(import-notation)